dwisc.py

```diff
@@ -85,6 +85,9 @@ def main(args):
             params['anneal_schedule'] = args.anneal_schedule
         else:
             params['annealing_time'] = args.annealing_time
+    
+        if args.h_gain_schedule != None:
+            params['h_gain_schedule'] = args.h_gain_schedule
```

Comment:
Did you test the case of just an `h_gain_schedule` schedule and no `anneal_schedule`?  I think in that case the hardware will give an error like "can't specify both h_gain_schedule and annealing_time".